skins/standard/global.css

```diff
@@ -35,7 +35,8 @@
     }
     /* monospace is much smaller than Verdana by default, so we make it a bit bigger. */
     pre, code, kbd {
-        font-size: medium;
+        font-family: "Droid Sans Mono", Menlo, Monaco, "Courier New", Courier, monospace;
+        font-size: small;
```

Comment:
Why is the font size of the User Story being reduced as part of changing the font, when one of the listed permitted fonts is the original font?